docs/os/build-details.md

```diff
@@ -15,7 +15,7 @@ It performs the following steps:
 1. cloning the [`github.com/armbian/build`](https://github.com/armbian/build/) repo into `armbian/armbian-build/`, if it doesn't exist already
 1. copying over the following files from the host system into `armbian/armbian-build/userpatches/overlay/`:
     1. `armbian/base/`: scripts and configs included in the Armbian image
-    1. `build/`: contains all [built Go binaries](/docs/go-apps.md) and their associated `.service` files
+    1. `build/`: contains all [built Go binaries](go-apps.md) and their associated `.service` files
```

Comment:
Not sure that works out of the `/docs/os/` directory?

Reply:
Yeah, I think you're right. On the live site it seems that the URL is https://base.shiftcrypto.ch/go, so I've updated this accordingly. It seems to render as `http://localhost:4000/go` on local Jekyll, so I think the new updated link should work.